internal/core/application/operator_service.go

```diff
@@ -650,6 +654,7 @@ func (o *operatorService) ListMarket(
 				FixedBaseFee:  market.FixedFee.BaseFee,
 				FixedQuoteFee: market.FixedFee.QuoteFee,
 			},
+			Balance: *balance,
```

Comment:
shoudlnt getUnlockedBalanceForMarket return already the pointer to a balance struct?

Reply:
this is to dereference the pointer actually since Balance is of type `Balance`, not of type `*Balance`.